content_loader.py

```diff
@@ -0,0 +1,85 @@
+import logging
+import requests
+
+from abc import ABC, abstractmethod
+from typing import Any, Tuple
+
+import credentials
+
+logging.basicConfig(level=logging.INFO)
+logger = logging.getLogger(__name__)
+
+
+class ContentLoader(ABC):
+    @abstractmethod
+    def load(self) -> Any:
+        """
+        Abstract method to load content.
+        """
+        raise NotImplementedError("Subclasses must implement this method")
+
+
+class ContentLoadError(Exception):
+    def __init__(self, message: str):
+        """
+        Initializes the ContentLoadError with an error message.
+        :param message: The error message.
+        """
+        super().__init__(message)
+
+
+class QuoteLoader(ContentLoader):
+    def __init__(self, quote_url: str):
+        """
+        Initializes the QuoteLoader with a URL.
+        :param quote_url: The URL to fetch the quote from.
+        """
+        self.quote_url = quote_url
+
+    def load(self) -> Tuple[str, str]:
+        """
+        Loads a quote from the URL.
+        :return: A tuple containing the quote and the author.
+        :raises ContentLoadError: If there is an error loading the quote.
+        """
+        try:
+            response = requests.get(self.quote_url)
+            response.raise_for_status()
+            data = response.json()
+            quote = data.get("content")
+            author = data.get("author")
+            if quote is None or author is None:
```

Comment:
It is better to add this to the logger as well.

Reply:
fixed issues on another branch, Thanks